common/api/ecschema-metadata.api.md

```diff
@@ -2066,7 +2066,7 @@ export enum PropertyType {
 }
 
 // @beta (undocumented)
-export function propertyTypeToString(type: PropertyType): "PrimitiveProperty" | "StructProperty" | "StructArrayProperty" | "NavigationProperty" | "PrimitiveArrayProperty";
+export function propertyTypeToString(type: PropertyType): "PrimitiveProperty" | "NavigationProperty" | "StructProperty" | "StructArrayProperty" | "PrimitiveArrayProperty";

```

Comment:
You may need to revert these changes. extract-api is super-flaky about the ecschema-metadata package for some reason.

Reply:
last time this happened caleb recommended `rush install && rush clean && rush build && rush extract-api`, I just pushed the result of that and believe it should be fixed